guide/14-deep-learning/mistral_workflow_with_text_models.ipynb

```diff
@@ -0,0 +1,621 @@
+{
```

Comment:
<p>"zip" not to be mentioned.</p><p>This method saves the model weights and creates a Deep Learning Package (.dlpk).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>